tests/tests_default.yml

```diff
@@ -4,10 +4,10 @@
   hosts: all
   gather_facts: false  # test that role works in this case
   roles:
-    - linux-system-roles.template
+    - linux-system-roles.keylime_server
   tasks:
     - name: Check header for ansible_managed, fingerprint
       include_tasks: tasks/check_header.yml
       vars:
         __file: /etc/foo.conf
```

Comment:
This file doesn't exist.  Is there an actual keylime server file that can be used here?  Or will running the role with no arguments not create any files?

Reply:
This is wrong, let me fix it. After running the role, it will create/update `/etc/keylime.conf`, on systems with legacy Keylime (Fedora 36 and 37) and create/update a few drop-in snippet files under `/etc/keylime/{verifier.conf.d,registrar.conf.d}`

Reply:
OK, I updated that test. Let's see if it works as expected.